photutils/psf/tests/test_psf_photometry.py

```diff
@@ -18,6 +22,11 @@
 except ImportError:
     HAS_SCIPY = False
 
+if pytest.__version__ >= '2.8':
```

Comment:
I would suggest to use either minversion from astropy.utils or LooseVersion from distutils to do these kind of comparisons:

``minversion(pytest, '2.8')`` or  ``LooseVersion(pytest.__version__) > LooseVersion('2.8')``